src/cbmc/cbmc_parse_options.cpp

```diff
@@ -905,6 +906,12 @@ bool cbmc_parse_optionst::process_goto_program(
     {
       status() << "Performing a full slice" << eom;
       full_slicer(goto_functions, ns);
+
+      status() << "Performing a reachability slice" << eom;
+       if(cmdline.isset("property"))
```

Comment:
1) I think this should be done before doing a full slice, and possibly should always be done (irrespective of whether full-slice is used or not).
2) The indent is weird.